toolkits/slack/tests/test_users.py

```diff
@@ -0,0 +1,7 @@
+import pytest
+
+
+@pytest.fixture
```

Comment:
You have this a couple times. Let's put this into a conftest.py so you can use it for all the tests.

Reply:
actually, we need to mock the class in two different modules: chat and users. to have a single fixture, we'd need to parameterize the location on each test case. not good. better to have two fixtures, I moved it from conftest to each of the testing modules, with the respective class location.